### ISSUE TITLE
Get 1M events syncing from Mammoth and going into PouchDB in Tesla

### ISSUE THREAD

**Issue by aguynamedben** [aguynamedben/tesla#3]
Depends on #2

**Comment by aguynamedben** [comment]
I tested FuzzySearch to 1M records, good enough for now. We'll need to tune FuzzySearch, will make a separate ticket for that.
